lib/node_modules/@stdlib/blas/ext/base/dsnansum/benchmark/benchmark.native.js

```diff
@@ -48,17 +49,7 @@ var opts = {
 * @returns {Function} benchmark function
 */
 function createBenchmark( len ) {
-	var x;
-	var i;
-
-	x = new Float32Array( len );
-	for ( i = 0; i < x.length; i++ ) {
-		if ( randu() < 0.2 ) {
-			x[ i ] = NaN;
-		} else {
-			x[ i ] = ( randu()*10.0 ) - 20.0;
-		}
-	}
+	var x = filledarrayBy( len, 'float32', rand );
```

Comment:
The changes to this (and the other benchmarks) are not equivalent to the original.